src/core_plugins/kibana/public/discover/components/field_chooser/discover_field.js

```diff
@@ -95,6 +93,13 @@ app.directive('discoverField', function ($compile) {
         }
       };
 
+      $scope.onKeyUp = function onKeyUp(e, field) {
+        // Support keyboard accessibility by toggling display on ENTER or SPACE keypress.
+        if ([13, 32].includes(e.keyCode)) {
+          $scope.toggleDetails(field);
+        }
+      };
```

Comment:
Sounds good to me. How do you feel about `kbn-accessible-click`? I think it's good to emphasize that it's for aiding accessibility up-front (and I imagine we'll have other services and directives geared towards accessibility, so they'll be some cohesion here), and I think there's nice symmetry with `ng-click` since they both have the word "click" in them.